python/lsst/ts/externalscripts/make_love_stress_tests.py

```diff
@@ -0,0 +1,324 @@
+        * remotes   : a dict, with each item as
+            CSC_name[:index]: `lsst.ts.salobj.Remote`
+
+        Constructing a `salobj.Remote` is slow (DM-17904), so configuration
+        may take a 10s or 100s of seconds per CSC.
+        """
+        self.log.info("Configure started")
+
+        # set configurations
+        self.config = config
+
+        # get credentials
+        self.username = "user"
+        self.password = os.environ.get("USER_USER_PASS")
+        if self.password is None:
+            raise RuntimeError(
+                "Configuration failed: environment variable USER_USER_PASS not defined"
+            )
+
+        # construct remotes
+        remotes = dict()
+        for name_index in config.data:
+            name, index = salobj.name_to_name_index(name_index)
+            self.log.debug(f"Create remote {name}:{index}")
+            if name_index not in remotes:
+                remote = salobj.Remote(domain=self.domain, name=name, index=index)
+                remotes[name_index] = remote
+        self.remotes = remotes
+
+    async def run(self):
+        """Run script."""
+
+        self.log.info(f"Waiting for {len(self.remotes)} remotes to be ready")
+        await asyncio.gather(*[remote.start_task for remote in self.remotes.values()])
+
+        # Enable all CSCs
+        for remote in self.remotes.values():
+            await salobj.set_summary_state(remote=remote, state=salobj.State.ENABLED)
+
+        event_streams = dict()
+        telemetry_streams = dict()
+        for name_index in self.remotes:
+            name, index = salobj.name_to_name_index(name_index)
+            salinfo = salobj.SalInfo(self.domain, name)
+            try:
+                event_streams[name_index] = salinfo.event_names
+                telemetry_streams[name_index] = salinfo.telemetry_names
+            finally:
+                await salinfo.close()
+
+        # Create clients and listen to ws messages
+        self.log.info(
+            f"Waiting for {self.config.number_of_clients} Manager Clients to be ready"
+        )
+        loop = asyncio.get_event_loop()
+        for i in range(self.config.number_of_clients):
+            self.clients.append(
+                ManagerClient(
+                    self.config.location,
+                    self.username,
+                    self.password,
+                    event_streams,
+                    telemetry_streams,
+                )
+            )
+
+        for client in self.clients:
+            loop.create_task(client.start_ws_client())
+
+        msg_count = 0
+        while msg_count < self.config.number_of_messages:
+            await asyncio.sleep(1)
+            new_count = 0
+            for client in self.clients:
+                new_count += len(client.msg_traces)
+            msg_count += new_count
+        mean_latency = round(self.get_mean_latency(), 2)
+        self.log.info(f"mean_latency_ms={mean_latency} num_messages={msg_count}")
```

Comment:
Might be beneficial to add some text to facilitate mining this particular data. Something like:


```
self.log.info(f"LOVE stress test result: mean_latency_ms={mean_latency} num_messages={msg_count}")
```